src/AzureDeprecation.Tests.Unit/AutoMapperTests.cs

```diff
@@ -83,6 +96,49 @@ public void AutoMapper_MapNewAzureDeprecationV1MessageToDeprecationInfo_IsValid(
             HasSameTimeline(deprecationInfo, publishedNotice);
         }
 
+        [Fact]
+        public void AutoMapper_MapEntityToPresentationModel_AllPropertiesMapped()
+        {
+            var fixture = new Fixture();
+            var dbEntity = fixture.Create<NoticeEntity>();
+
+            var resultModel = _mapper.Map<AzureDeprecation.APIs.REST.Contracts.DeprecationInfo>(dbEntity);
+            
+            resultModel.DeprecationData
+                .WithDeepEqual(dbEntity.DeprecationInfo)
+                .Assert();
+            
+            resultModel
+                .WithDeepEqual(dbEntity)
+                .IgnoreDestinationProperty(x => x.DeprecationInfo)
+                .IgnoreSourceProperty(x => x.DeprecationData)
+                .Assert();
```

Comment:
I don't have a problem with fluent assertions but the rest of the tests don't use them. Would you mind aligning them with existing tests please?

Reply:
The problem is that I use DeepEqual package that provide only this "fluent like" methods. If it's ok, I'd prefer to leave it as it is now

Reply:
And why do we need DeepEqual exactly?

Reply:
I don't prefer magical things to simplify things but let's just go ahead